src/request-schema.js

```diff
@@ -0,0 +1,49 @@
+const dynamoose = require('dynamoose')
+
+dynamoose.setDefaults({
+  create: false,
+  waitForActiveTimeout: 5000
+})
+
+const Schema = dynamoose.Schema
+
+const requestSchema = new Schema({
+  request_id: {
+    type: String,
+    hashKey: true
+  },
+  user_primary_id: String,
+  request_type: String,
+  request_sub_type: {
```

Comment:
Can you explain what this formatting does?

Reply:
The intention was to define an object with the props 'value' and 'desc' on it, as they are in Alma. I've tested this and this isn't the correct way to do this in dynamoose, I'll update it now